ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/FlowAutoConfiguration.java

```diff
@@ -50,19 +51,35 @@ public void registerInputBeans() {
 	}
 
 	public ComponentExecutor componentExecutor(final PipelineConfig pipelineConfig) {
-		List<LdiTransformer> ldiTransformers = pipelineConfig.getTransformers()
+		List<LdioTransformer> ldioTransformers = pipelineConfig.getTransformers()
 				.stream()
 				.map(this::getLdioTransformer)
 				.toList();
+
 		List<LdiOutput> ldiOutputs = pipelineConfig.getOutputs()
 				.stream()
 				.map(this::getLdioOutput)
 				.toList();
 
 		LdiSender ldiSender = new LdiSender(eventPublisher, ldiOutputs);
+
+		List<LdioTransformer> processorChain = new ArrayList<>();
+
+		if (!ldioTransformers.isEmpty()) {
+			processorChain.addAll(ldioTransformers.subList(0, ldioTransformers.size()));
```

Comment:
Why do you use a subList here? Is the empty check required, if you add an empty list with addAll, don't you still end up with an empty list?